pkgs/applications/video/mpv/scripts/buildLua.nix

```diff
@@ -2,7 +2,6 @@
 , stdenvNoCC }:
 
 let
-  inherit (lib) hasPrefix hasSuffix removeSuffix;
```

Comment:
i'd not make unrelated changes in this PR.

Reply:
Okay, removed.